index.html

```diff
@@ -51,89 +59,103 @@ <h1 class="title">I'm Dhiraj<br />Glad to see<br />you!</h1>
+            </ul>
+            <button class="btn">See Project</button>
+          </div>
         </div>
-        <div class="card">
+        <!-- card-4 -->
+        <div class="card card-reverse">
           <img src="images/Project-4.png" alt="Project-4" />
-          <h2 class="title">Multi-Post Stories<br />Stories</h2>
-          <ul class="info">
-            <li class="info-items">CANOPY</li>
-            <li class="info-item"><i class="fa-solid fa-circle fa-xs"></i></li>
-            <li class="info-items">Back End Dev</li>
-            <li class="info-item"><i class="fa-solid fa-circle fa-xs"></i></li>
-            <li class="info-items">2015</li>
-          </ul>
-          <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="program">
-            <li class="language">html</li>
-            <li class="language">css</li>
-            <li class="language">javascript</li>
-          </ul>
-          <button class="btn">See Project</button>
+          <div class="card-info-container">
+            <h2 class="title">Uber Navigation</h2>
+            <ul class="info">
+              <li class="info-items">CANOPY</li>
+              <li class="info-item"><i class="fa-solid fa-circle fa-xs"></i></li>
+              <li class="info-items">Back End Dev</li>
+              <li class="info-item"><i class="fa-solid fa-circle fa-xs"></i></li>
+              <li class="info-items">2015</li>
+            </ul>
+            <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="program">
+              <li class="language">html</li>
+              <li class="language">css</li>
+              <li class="language">javascript</li>
+            </ul>
+            <button class="btn">See Project</button>
+          </div>
         </div>
       </section>
-      <section class="about container-padding-x container">
-        <div class="about-social-container">
-          <h2 class="title">About<br />Myself</h2>
-          <p class="text">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
-            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
-            me.
-          </p>
+      <!-- About Section -->
+      <section id="about" class="about container-padding-x container">
+        <div class="social-container">
           <div class="about-social-container">
+            <h2 class="title">About<br />Myself</h2>
+            <p class="text">
+              Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+              me.
+            </p>
```

Comment:
- [ ] Good job for this job you did :+1: , however , make sure the  ` about section` fill all the width of the screen in the `desktop` version as it is represented in the Figma design. Kindly remove the space in `Red`. :100: on the picture below

Your design 

![Screenshot from 2022-12-08 23-11-43](https://user-images.githubusercontent.com/80458199/206574781-cdd675ec-cfc8-431f-bbe5-50b2c6700c8d.png)